pkg/datasource/compat/wrapper.go

```diff
@@ -259,6 +260,20 @@ func WrapAccessors(source datasource.DataSource, mntnsidAccessor datasource.Fiel
 			datasource.ColumnsMaxWidthAnnotation: "64",
 		}),
 		datasource.WithFlags(datasource.FieldFlagHidden),
+		datasource.WithOrder(-25),
+	)
+	if err != nil {
+		return nil, err
+	}
+	ev.containerpidAccessor, err = runtime.AddSubField(
+		"containerPid",
+		api.Kind_Uint32,
+		datasource.WithAnnotations(map[string]string{
+			datasource.ColumnsWidthAnnotation:     "6",
+			datasource.ColumnsAlignmentAnnotation: "right",
+			datasource.ColumnsMaxWidthAnnotation:  "64",
```

Comment:
Does `"template": "containerPid"` work to avoid repeating it again?

Reply:
It seems to work. So I can remove the hidden flag because it automatically comes from the template.